src/site/assets/js/execute-axe-check.js

```diff
@@ -0,0 +1,78 @@
+function processAxeCheckResults(error, results) {
+  if (error) {
+    console.log('Error executing the Axe check!');
+    console.log(error);
+    return;
+  } else if (results.violations.length === 0) {
+    return;
+  }
+
+  var bannerEl = document.createElement('div');
+
+  var details = '<details class="vads-u-background-color--gold-lighter vads-u-padding--1">';
+  details += '<summary>There are (' + results.violations.length + ') accessibility issues on this page.</summary>';
+
+  var violationsList = '<ul class="usa-unstyled-list vads-u-padding-y--2 vads-u-padding-x--6">';
```

Comment:
Can we add two classes to the UL? ` vads-u-border-color--primary-darker vads-u-border-top--1px`